webview/src/experiments/components/table/CellRowActions.tsx

```diff
@@ -128,11 +152,14 @@ export const CellRowActions: React.FC<CellRowActionsProps> = ({
           showSubRowStates={showSubRowStates}
           subRowsAffected={plotSelections}
           testId={'row-action-plot'}
-          tooltipContent={getTooltipContent(
-            !!bulletColor,
-            'plot',
-            'To open the plots view click the plot icon in the top left corner\nor use "DVC: Show Plots" from the command palette.'
-          )}
+          tooltipOffset={isWorkspace ? [0, -16] : undefined}
```

Comment:
[F] This is the smallest offset that enables the tooltip to be reachable from the workspace row.

Reply:
Otherwise it disappears before you can mouse into it.